### PR TITLE
docs for bulk role change in web-sdk

### PR DESCRIPTION
**Pull request by amar-1995**

### CODE REVIEW

File: docs/javascript/v2/features/change-role.mdx [hunk 1]
```diff
@@ -33,20 +33,45 @@ console.log("can I end change role - ", permissions.changeRole);
 const { video, audio, screen } = hmsStore.getState(selectIsAllowedToPublish);
 ```
 
-## Change Role API
+## Change Role APIs
+
+There are two way of chaning peer's role using our api:
+1. [`Single Peer Role API`](#change-single-peer-role-api) - It is useful when we need to change role of a single peer.
+2. [`Bulk Peer Role API`](#change-bulk-peer-role-api) - We could change a role of all the peer to another role. It will be helpful to make breakout rooms, waiting rooms, etc.
```
Comment (raviteja83) [inline docs/javascript/v2/features/change-role.mdx:40]: all peers of particular roles to a given role. we should also mention that permissions are to be enabled in dashboard.

File: docs/javascript/v2/features/change-role.mdx [hunk 2]
```diff
@@ -33,20 +33,45 @@ console.log("can I end change role - ", permissions.changeRole);
...
 - `force`(optional): The other peer gets a request by default to accept a role change, by setting force to true, the role can be changed
   without requesting for approval.
 
+
+### Change Bulk Peer Role API
+
+If the local peer has permissions to change role(`permissions.changeRole`), they can convert all roles from a list of roles, to another role using
+the below interface.
+
+```js
```
Comment (raviteja83) [inline docs/javascript/v2/features/change-role.mdx:62]: we should have both react and js snippets even though this is straightforward for us.

File: docs/javascript/v2/features/change-role.mdx (hunk 1 above)
Comment (raviteja83) [reply docs/javascript/v2/features/change-role.mdx:40]: can we also update the screenshot in this page.

Comment (amar-1995) [reply docs/javascript/v2/features/change-role.mdx:40]: Which screenshot?

File: docs/javascript/v2/features/change-role.mdx (hunk 2 above)
Comment (amar-1995) [reply docs/javascript/v2/features/change-role.mdx:62]: Do we need to add react snippet in line 63? @Nixon-Samuel

Comment (Nixon-Samuel) [reply docs/javascript/v2/features/change-role.mdx:62]: yes @amar-1995 Would be easy for customers to adopt.

Comment (amar-1995) [reply docs/javascript/v2/features/change-role.mdx:85]: done

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**